common/types/pb/type.go

```diff
@@ -564,8 +564,10 @@ func zeroValueOf(msg proto.Message) proto.Message {
 }
 
 var (
+	jsonValueTypeURL = "types.googleapis.com/google.protobuf.Value"
```

Comment:
This is just to unmarshal an empty `any{}` into JSON's NullValue right?

Reply:
That's right. It was missing before and so an unset any field was actually erroring on access. It would be a good conformance test case.